ros/src/tl_detector/tl_detector.py

```diff
@@ -34,9 +34,14 @@ def __init__(self):
         self.last_state = TrafficLight.UNKNOWN
         self.last_wp = -1
         self.state_count = 0
+        self.image_counter = 0
 
         self.bridge = CvBridge()
-        self.light_classifier = TLClassifier("frozen_inference_graph.pb")
+        if not SAVING_IMAGES:
+            self.light_classifier = TLClassifier("frozen_inference_graph.pb")
+        else:
+            if not os.path.exists("../../../images/"):
```

Comment:
Let's keep the _folder_ location ("../../../images/") as a config param too...so we don't need to recompile, and we have one place to control these values.